jdaviz/configs/specviz/plugins/line_analysis/line_analysis.py

```diff
@@ -361,11 +366,16 @@ def _uncertainty(result):
                                                                        equivalencies=u.spectral()),
                         flux=spec_subtracted.flux,
                         uncertainty=spec_subtracted.uncertainty)
+                    raw_result = analysis.line_flux(wave_spec)
                     # When flux is equivalent to Jy, lineflux result should be shown in W/m2
                     if flux_unit.is_equivalent(u.Unit('W/m2/m'/u.sr)):
-                        temp_result = analysis.line_flux(wave_spec).to('W/(m2sr)')
+                        final_unit = u.Unit('W/m2/sr')
                     else:
-                        temp_result = analysis.line_flux(wave_spec).to('W/m2')
+                        final_unit = u.Unit('W/m2')
+
+                    temp_result = raw_result.to(final_unit)
+                    if getattr(raw_result, 'uncertainty', None) is not None:
+                        temp_result.uncertainty = raw_result.uncertainty.to(final_unit)
```

Comment:
hopefully this will be improved upstream some day... but does the trick for now!

Reply:
I created https://github.com/astropy/specutils/issues/980 in specutils to make sure we don't forget to discuss

Reply:
my comment on these lines was referring to the fact that `.uncertainty` (or any arbitrary attribute added to a quantity) isn't supported by astropy and so is stripped from the quantity object when calling `.to` (see https://github.com/astropy/specutils/pull/961 for some related discussion).

Reply:
Ahh right. Sorry my comment was meant as a direct response to your general review. Sorry, poor placement